ncm-spma/src/main/pan/components/spma/yumng/schema.pan

```diff
@@ -0,0 +1,73 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template components/spma/yumng/schema;
+
+include 'components/spma/schema';
+
+type SOFTWARE_GROUP = {
+    "default" : boolean = true
+    "mandatory" : boolean = true
+    "optional" : boolean = false
+    "names" : string[]
```

Comment:
aha - if yumng can have its own schema then we can drop this and revert to the code we actually run where /software/groups is a list not a dict. I will send a patch for that tomorrow.